plugin.py

```diff
@@ -108,6 +108,38 @@ def install_or_update(cls) -> None:
             shutil.rmtree(cls.basedir(), ignore_errors=True)
             raise
 
+    def on_pre_server_command(self, command: Mapping[str, Any], done_callback: Callable[[], None]) -> bool:
+
+        if len(command["arguments"]) == 0:
+            return False
+
+        output = command["arguments"][0]
```

Comment:
Wouldn't you want to do that only for specific commands (check `command['command']`)?. Doing this for all commands that have arguments seems a bit too broad to me.

Reply:
Fair I'll write logic to check if they involve cargo

Reply:
Doesn't seem resolved?